cluster-autoscaler/cloudprovider/mcm/mcm_cloud_provider.go

```diff
@@ -195,6 +193,19 @@ func (mcm *mcmCloudProvider) GetResourceLimiter() (*cloudprovider.ResourceLimite
 // Refresh is called before every main loop and can be used to dynamically update cloud provider state.
 // In particular the list of node groups returned by NodeGroups can change as a result of CloudProvider.Refresh().
 func (mcm *mcmCloudProvider) Refresh() error {
+
+	namespace := mcm.mcmManager.namespace
+	deployment, err := mcm.mcmManager.deploymentLister.Deployments(namespace).Get("machine-controller-manager")
+	if err != nil {
+		klog.Errorf("failed to get machine-controller-manager deployment: %v", err.Error())
+		return err
```

Comment:
```suggestion
                return fmt.Errorf("failed to get machine-controller-manager deployment: %v", err)
```
No need to log it here as the caller is logging the error